testing/test-cases/selenium-tests/glContour/testGlContour.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import urllib
+from selenium_test import FirefoxTest, ChromeTest,\
+    setUpModule, tearDownModule
+
+
+class glContourBase(object):
+    testCase = ('glContour',)
+    testRevision = 2
```

Comment:
Midas revisions don't work as I originally thought.  (You can tell it which revision you are uploading.)  So when you ran it the first time, it set revision 1, but the test is looking for revision 2.  I believe if you repeated the upload, it would work.  This part of the upload is clunky which is why I generally tell people I will do it myself ;)... I eventually want to improve this when I transition over to girder.